### PR TITLE
DOC: Make pyplot import explicit in the 10 minutes to pandas page

### PR DESCRIPTION
**Pull request by yuseitahara**
Beginners don't know what is `plt`, thus it is better to show how to import it explicitly.

This is a continuation of https://github.com/pandas-dev/pandas/pull/30274

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by yuseitahara** [comment]
@datapythonista Could you please merge this PR? Thank you.

**Review by datapythonista — APPROVED** [review]
Lgtm, thanks!

File: doc/source/getting_started/10min.rst [hunk 1]
```diff
@@ -717,6 +711,8 @@ of the columns with labels:
 
 .. ipython:: python
 
+   import matplotlib.pyplot as plt
```
Comment (jreback) [inline doc/source/getting_started/10min.rst]: I would leave this where it was, and maybe even add a comment (removing the suppress is ok)

**Review by jreback — CHANGES_REQUESTED** [review]

File: doc/source/getting_started/10min.rst (hunk 1 above)
Comment (yuseitahara) [reply doc/source/getting_started/10min.rst]: @jreback Thank you for your comment. I moved the import statement to the previous position and added a comment.

**Comment by yuseitahara** [comment]
> AttributeError: module 'IPython.utils.py3compat' has no attribute 'cast_bytes_py2'

pandas-dev.pandas (Web_and_Docs) is failing, but is it related to my change?

https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=24127&view=logs&j=5291ec4e-9543-503b-43ce-895cf0e06ba3&t=ba0a2b35-e1f8-54df-fd66-2b9fde6f4b1f&l=5158

**Comment by rushabh-v** [comment]
>pandas-dev.pandas (Web_and_Docs) is failing, but is it related to my change?

I also have the same problem in my PR. see #30282

**Comment by datapythonista** [comment]
@yuseitahara can you merge master into your branch please? It should fix the CI.

**Comment by datapythonista** [comment]
@jreback you're changes have been addressed, all green here.

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @yuseitahara

**Pull request merged**